web-react-ts/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Once we get this merged I'll make a docs ticket to updated the readme to match the other CRA readme